auxx/src/HashInstaller.hs

```diff
@@ -0,0 +1,14 @@
+module Main where
+
+import qualified Data.ByteString.Lazy as BSL
+import           Formatting           (sformat)
+import           Pos.Crypto               (hashHexF)
```

Comment:
Why did the spacing end up weird here even after using stylish? Maybe there's a tab.

Reply:
i think thats a refactor i did after stylish, re-running it